Core/src/org/sleuthkit/autopsy/translation/TranslatedTextViewer.java

```diff
@@ -0,0 +1,425 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2019 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.translation;
```

Comment:
Is there a particular reason we have made a new translation package for this content viewer? Having both a texttranslation and translation package may be confusing. Could this be moved to contentviewers.textcontentviewer package?

Reply:
The reasons are:
1. I didn't see the texttranslation package when I moved it 
2. We don't want to export these classes so it cant go into the TextTranslation package (not that you were suggesting that)
3. There is more translation stuff coming in which doesn't really have anything to do with the TextContentViewer but keeping the translations stuff together would be ideal in my opinion

The first one there is clearly the main reason and is my bad. In the next story which I am still working on and requires moving more of the translations stuff into Autopsy I have remedied this  concern slightly by eliminating this new translation package and instead adding a new org.sleuthkit.autopsy.texttranslation.ui package for the translation stuff being added as it is all UI related stuff.